deploy-manage/deploy/cloud-enterprise/deploy-large-installation.md

```diff
@@ -27,6 +27,10 @@ This type of installation is recommended for deployments with significant overal
 :alt: A large installation with nine to twelve hosts across three availability zones
 :::
 
+::::{note}
+The host with the allocator role greyed out represents the first host to be installed. This host temporarily holds all roles until the other nodes are added and configured. Eventually, the allocator role will be removed from this host.
```

Comment:
```suggestion
In the diagram, the Director Coordinator host in Availability zone 1 has the allocator role greyed out. This host temporarily holds all roles until the other nodes are added and configured. Eventually, the allocator role will be removed from this host.
```

I'd rephrase this a bit for accessibility, since people reading using a screen reader can't access the diagram, and others can see the colour variances.

Reply:
Ok, I'll rephrase it then, because I want to keep that the host represents the first host to be installed, as that information is not provided and it's relevant.